circleci-orb/BUILD.bazel

```
@@ -0,0 +1,20 @@
+ORB_NAME = "devversion/ng-dev-infra-private"
```

```suggestion
ORB_NAME = "angular/dev-infra"
```

I think we can actually host this as a private orb which means that we can just name it dev-infra?

Or maybe it makes sense to still use `dev-infra-private`

---

Yeah, I think this makes sense. I wasn't sure if I had access, but now I surely have, and it seems to work!